source/math/FloatBox2D.ooc

```diff
@@ -63,6 +63,18 @@ FloatBox2D: cover {
 	shrink: func ~fraction (margin: Float) -> This {
 		this pad(-margin * this height / 2.0f)
 	}
+	resizeTo: func (size: FloatSize2D) -> This {
+		This createAround(this center, size)
+	}
+	scale: func (value: Float) -> This {
+		This createAround(this center, value * this size)
+	}
+	enlargeTo: func (size: FloatSize2D) -> This {
+		This createAround(this center, this size + ((size - this size) maximum~Float(0.0f)))
+	}
+	shrinkTo: func (size: FloatSize2D) -> This {
+		This createAround(this center, this size + ((size - this size) minimum~Float(0.0f)))
+	}
```

Comment:
I think this would become more readable if we changed `enlargeTo` and `shrinkTo` to
```ooc
This createAround(this center, FloatSize2D maximum(this size, size))
```
and
```ooc
This createAround(this center, FloatSize2D minimum(this size, size))
```
respectively.